### PR TITLE
Styling 2 15

### PR DESCRIPTION
**Pull request by AliSchlereth**
A few little things:
-Flash messages now are closable.
-Affiliations/groups are listed as a table row instead of a non -labeled bulleted list.
-Tables are sortable by each column.
-When a user signs up they are automatically signed in.
-Updated CanCanCan to allow enrolled users to edit their groups
--If we don't want enrolled users to do this, we should update the enrolled users's page so that it doesn't show an update affiliations button. I think there might need to be a test update. 
-Update a few of the registration tests to pass. They were auto registering as staff and enrolled which is no longer how we invite users. 

@bradgreen3 @JStans12

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -21,7 +21,11 @@ def create
 
   # DELETE /resource/sign_out
   def destroy
-    super
```
Comment (JStans12) [inline app/controllers/users/sessions_controller.rb]: not so super?

File: app/views/admin/roles/users/edit.html.erb [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
-    <table id="user-role-search" class="table table-hover role-table">
+    <table id="user-role-search" class="table table-hover role-table" data-toggle="table">
       <thead>
         <tr>
           <th></th>
-          <th>First</th>
-          <th>Last</th>
-          <th>Cohort</th>
-          <th>Roles</th>
+          <th data-sortable="true">First</th>
```
Comment (JStans12) [inline app/views/admin/roles/users/edit.html.erb]: 👍

File: Gemfile [hunk 3]
```diff
@@ -32,6 +32,7 @@ gem 'cancancan'
 # Styling
 gem 'bootstrap-sass'
 gem 'rack-cors', require: 'rack/cors'
+gem "bootstrap-table-rails"
```
Comment (bradgreen3) [inline Gemfile]: Sweet gem

File: app/controllers/users/registrations_controller.rb [hunk 4]
```diff
@@ -29,8 +29,15 @@ def create
       @user.skip_confirmation!
       if @user.save
         session[:invitation_code] = nil
-        flash[:success] = 'You have succesfully signed up! Please log in to continue.'
-        redirect_to new_user_session_path
+        flash[:success] = 'You have succesfully signed up!'
+        # redirect_to new_user_session_path
```
Comment (bradgreen3) [inline app/controllers/users/registrations_controller.rb]: Might want to remove these comments

File: app/controllers/users/sessions_controller.rb (hunk 1 above)
Comment (bradgreen3) [inline app/controllers/users/sessions_controller.rb]: Sad

File: app/views/affiliations/new.html.erb [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<div class="container">
+<div class="container col-md-4 col-md-offset-4">
```
Comment (bradgreen3) [inline app/views/affiliations/new.html.erb]: Ah, sensical page placement!

**Pull request merged**